libs/filamat/src/sca/GLSLTools.cpp

```diff
@@ -359,4 +359,12 @@ void GLSLTools::prepareShaderParser(MaterialBuilder::TargetApi targetApi, glslan
     }
 }
 
+void GLSLTools::textureLodBias(TShader& shader) {
+    TIntermediate* intermediate = shader.getIntermediate();
+    TIntermNode* root = intermediate->getTreeRoot();
+    ASTUtils::textureLodBias(intermediate, root,
+            "material(struct-MaterialInputs",
```

Comment:
Why can't we put the full signature here? Also I'm confused by the `struct-` but I'm guessing that's a glslang thing

Reply:
The signature changes when we change MaterialInputs :(( It's apparently something like struct-MaterialInputs-v4f-v1f-etc. Apparently the struct flattened??

Reply:
It's because the signature seems to change based on what is inside `MaterialInputs` and yes the `struct` is part of the signature, apparently.